### ISSUE TITLE
Embed in Medium

### ISSUE THREAD

**Issue by nitanagdeote** [datavis-tech/vizhub-issue-tracker#481]
I share the link from  vizhub on medium, it did not work. Looks like this 

![embedVizhub](https://user-images.githubusercontent.com/13540669/94747847-edaf8400-034d-11eb-9e5c-8205af1692b3.png)

**Comment by curran** [comment]
To make this work we need to implement https://embed.ly/ .

@stushurik and I did this work long ago for the original datavis.tech, and it worked! We could embed things from datavis.tech into Medium. The same work needs to be done now for VizHub.

From what I recall, we need to implement OEmbed, and some other things, then submit something to Embedly, then wait for 2 weeks for it to be approved.

First we need to release a first working version of Embed to production. Currently it's only in staging.

**Comment by curran** [comment]
Blocked by https://github.com/datavis-tech/vizhub-issue-tracker/issues/472

**Comment by curran** [comment]
More details here: https://embed.ly/providers/new

**Comment by curran** [comment]
Previous implmenentation: https://github.com/datavis-tech/datavis-tech-app-legacy/search?q=oembed

**Comment by curran** [comment]
Status:
 * oEmbed implementation is done and deployed to production.
 * Embedly submission has been done (today).
 * Awaiting approval/response from Embedly (should be within 2 weeks)

**Comment by curran** [comment]
Great news - Embedly application has been approved! This means VizHub embeds will start working soon in Medium.

![image](https://user-images.githubusercontent.com/68416/102394343-9e214180-3fa7-11eb-9bd9-01be088f76eb.png)

I'll close this issue when I see it actually working in Medium. We need to wait a bit more I guess.

**Comment by curran** [comment]
This is working now woohoo!

https://currankelleher.medium.com/vizhub-embed-testing-3cc46a04dec

New issue for followup: #678
